hudi-client/src/test/java/org/apache/hudi/index/TestHoodieIndex.java

```diff
@@ -43,9 +44,11 @@ public void setUp() throws Exception {
   }
 
   @After
-  public void tearDown() {
+  public void tearDown() throws IOException {
     cleanupSparkContexts();
-    cleanupMetaClient();
+    cleanupFileSystem();
```

Comment:
cleanupFileSystem and cleanupTestDataGenerator was added as part of SimpleHoodieIndex. These are not required as per 0.5.3.